source/math/Quaternion.ooc

```diff
@@ -99,17 +187,39 @@ Quaternion: cover {
 		this y ==  y &&
 		this z == other z
 	}
+	/*operator + (other: This) -> This {
+		This new (this w + other w, this x + other x, this y + other y, this z + other z)
+	}*/
 	operator + (other: This) -> This {
-		This new(this w + other w, this x + other x, this y + other y, this z + other z)
+		This new (this real + other real, this imaginary + other imaginary)
 	}
+	/*operator - (other: This) -> This {
+		This new (this w - other w, this x - other x, this y - other y, this z - other z)
+	}*/
```

Comment:
Did not want to remove it until I was sure it worked - I am not used to this whole Git thing ;-)